sdks/java/io/file-schema-transform/src/main/java/org/apache/beam/sdk/io/fileschematransform/FileWriteSchemaTransformConfiguration.java

```diff
@@ -51,43 +52,41 @@ public static XmlConfiguration.Builder xmlConfigurationBuilder() {
         .setCharset(StandardCharsets.UTF_8.name());
   }
 
-  /**
-   * The format of the file content. Used as String key lookup of {@link
-   * FileWriteSchemaTransformFormatProviders#loadProviders()}.
-   */
+  @SchemaFieldDescription(
+      "The format of the file content. Value must be one of: \"avro\", \"csv\", \"json\", \"parquet\", \"xml\"")
   public abstract String getFormat();
 
-  /** A common prefix to use for all generated filenames. */
+  @SchemaFieldDescription("A common prefix to use for all generated filenames.")
   public abstract String getFilenamePrefix();
 
-  /**
-   * The compression of all generated shard files. By default, appends the respective extension to
-   * the filename. See {@link org.apache.beam.sdk.io.Compression} for expected values.
-   */
+  /** See {@link org.apache.beam.sdk.io.Compression} for expected values. */
+  @SchemaFieldDescription(
+      "The compression of all generated shard files. By default, appends the respective extension to the filename.")
```

Comment:
Can you add a link to https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/Compression.html for expected inputs?